test/e2e/storage/external/testdata/example.yaml

```diff
@@ -0,0 +1,19 @@
+StorageClass:
+  FromExistingClassName: example
+DriverInfo:
+  Name: example
+  RequiredAccessModes:
+  - ReadWriteOnce
+  Capabilities:
+    persistence: true
+    multipods: true
+    exec: true
+    block: true
+    fsGroup: true
+    topology: true
+    controllerExpansion: true
+    nodeExpansion: true
+    volumeLimits: false
+  StressTestOptions:
+    NumPods: 10
+    NumRestarts: 20
```

Comment:
Can you include `NumSnapshots: 10`? (Or some reasonable value), this will help users to run the volume snapshot stress test.

Reply:
Done